backend/import/do_import.py

```diff
@@ -1,3 +1,4 @@
+"""Import org units and employees from MO."""
```

Comment:
The doc string normally goes _below_ any leading comment and imports IMO.

Reply:
PEP 257 says that a docstring "is a string literal that occurs as the first statement in a module, function, class, or method definition".

As imports are statements, this means it should go _before_ those. However, it's not a problem to put it after the license boilerplate.